doc/Command-Reference.md

```diff
@@ -6169,6 +6171,52 @@ This command displays the user persistet-watermark for the queues (Egress shared
   admin@sonic:~$ sonic-clear priority-group persistent-watermark headroom
   ```
 
+#### Buffer Pool
+
+This sub-section explains the following buffer-pool parameters that can be displayed using "show buffer_pool" command.
+1) buffer_pool watermark
+2) buffer_pool persistent-watermark
+
+**show buffer_pool watermark**
+
+This command displays the user watermark for all the buffer_pools(Total Buffer pool maximum occupancy per pool)
```

Comment:
please remove the description within the brackets. I think the output explains it well. Also change 'buffer_pools' to 'buffer pools'